### DISCUSSION TITLE
Rapidly changing zsh pid

### DISCUSSION THREAD

**Discussion by poetaman** [zdharma-continuum/zinit#183] [Q&A]
Not sure what is causing this, but I see rapidly changing zsh pids. I'll begin here instead of checking with tmux. I have 10 tmux sessions open Here's a video from htop:

https://user-images.githubusercontent.com/71736629/154766551-dfa8af54-26b2-4106-b220-8b38ccbb65c5.mp4

**Comment by pschmitt** [comment]
That's probably the zinit-scheduler doing its job. Or a plugin.

**Comment by vladdoster** [comment]
What exactly is the issue? Not sure what to make of your just an observation.
